### ISSUE TITLE
Work thumbnail images lack alt text

### ISSUE THREAD

**Issue by noraegloff** [LafayetteCollegeLibraries/spot#853]
Works viewed in Collection, Catalog and result-set views lack alt text. 
`<img>` elements must have either alternate text or a role of none or presentation. 

https://ldr.lafayette.edu/collections/lafayette-newspaper-archive and https://ldr.lafayette.edu/collections/historical-photos were the examples to which testing tools were applied, but this issue applies to all collections-level landing pages.

Example elements/source code viewed in test: 
```
<img class="hidden-xs file_listing_thumbnail" src="/downloads/rx913q00j?file=thumbnail">
<img class="hidden-xs file_listing_thumbnail" src="/downloads/m613mx805?file=thumbnail">
<img class="hidden-xs file_listing_thumbnail" src="/downloads/dz010q19j?file=thumbnail">
<img class="hidden-xs file_listing_thumbnail" src="/downloads/h989r3467?file=thumbnail">
<img class="hidden-xs file_listing_thumbnail" src="/downloads/s1784k99j?file=thumbnail">
<img class="hidden-xs file_listing_thumbnail" src="/downloads/v979v3478?file=thumbnail">
<img class="hidden-xs file_listing_thumbnail" src="/downloads/b8515n62b?file=thumbnail">
<img class="hidden-xs file_listing_thumbnail" src="/downloads/k0698770z?file=thumbnail">
<img class="hidden-xs file_listing_thumbnail" src="/downloads/jh343s66t?file=thumbnail">
<img class="hidden-xs file_listing_thumbnail" src="/downloads/qj72p747m?file=thumbnail">
```

**Comment by noraegloff** [comment]
Deque aXe tools severity level = **critical**

**Comment by rococodogs** [comment]
see #857, this is fixed in hyrax v3.
